server/aws-lsp-yaml-json/README.md

```diff
@@ -1,3 +1,9 @@
+## Exports
+── index
+    └── CreateYamlJsonLanguageServer - a function that creates and returns a new instance of DEXP Language Server
```

Comment:
Have it already been decided whether we distribute the json and yaml servers as one combined package?

Reply:
not yet, but based on my performance testing I think it's not necessary https://quip-amazon.com/4AluAbChYCdz/LSP-testing-Analysis-and-results